src/app.js

```diff
@@ -54,10 +55,27 @@ async function listAccessKeys(action,settings) {
     });
 }
 
+async function listUsers(action,settings) {
```

Comment:
Why `async` if we don't use `await`?

Let's make use of it anyway.
You could call `.promise()` on the method's return value to get a promise.
Please wrap it with a try-catch block and throw an error describing what AWS method fails.

https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/using-promises.html

Reply:
fixed